addon/usables/use-machine.js

```diff
@@ -0,0 +1,115 @@
+import { interpret, createMachine, StateNode } from 'xstate';
+import { setUsableManager } from 'ember-usable';
+import { later, cancel } from '@ember/runloop';
+import { tracked } from '@glimmer/tracking';
+import { getOwner, setOwner } from '@ember/application';
+import { DEBUG } from '@glimmer/env';
+import { warn } from '@ember/debug';
+
+export const ARGS_STATE_CHANGE_WARNING =
+  'A change to passed `args` or a local state change triggered an update to a `useMachine`-usable. This is currently unsupported and should probably be handled explicitly in your xstate-machine. If you feel this is necessary behavior that should be supported by `ember-statecharts` please raise this as an issue and provide an example of what you are trying to do.';
+
+export class InterpreterService {
+  @tracked service;
+  @tracked currentState;
+
+  constructor(machine, interpreterOptions) {
+    this.machine = machine;
+    this.interpreterOptions = interpreterOptions || {};
+  }
+
+  get state() {
+    return {
+      currentState: this.currentState,
```

Comment:
If we name this `state` it's aligned to `xstate/react` and `xstate/vue`

Reply:
maybe just.. `current`? since the getter is already `state`? we have the mental context that we're already getting the "state", but `current` would just mean the current version of it?

Reply:
> If we name this state it's aligned to `xstate/react` and 
`xstate/vue`

I was thinking the same as I saw that :+1:

Reply:
ah, ignore me. I thought this was used in a different way. I'm 👍 on `state`